clients/vault/src/oracle/testing_utils.rs

```diff
@@ -43,12 +44,11 @@ fn stellar_relay_config_abs_path(
 		.expect("should be able to extract config")
 }
 
-pub fn get_secret_key(with_currency: bool, is_mainnet: bool) -> String {
-	let suffix = if with_currency { "_with_currency" } else { "" };
-	let directory = if is_mainnet { "mainnet" } else { "testnet" };
-
-	let path = format!("./resources/secretkey/{directory}/stellar_secretkey_{directory}{suffix}");
-	std::fs::read_to_string(path).expect("should return a string")
+pub fn get_secret_key_from_env(with_currency: bool, is_mainnet: bool) -> String {
```

Comment:
Probably worth adding a comment here about what the `with_currency` flag actually does.

Reply:
Maybe the whole function name is a little unfortunate though and it's least confusing to split this into two functions:
- `get_secret_key_from_env_with_currency(is_mainnet: bool)` and 
- `get_secret_key_from_env(is_mainnet: bool)`

Reply:
Yes. Makes more sense and is a bit more readable.

Reply:
In fact, what if we directly use `get_source_secret_key_from_env` and `get_dest_secret_key_from_env`? Source being = `with_currency`. Would it be too confusing?

Reply:
Sure, we can also do that 👍 @b-yap do you have an opinion here?

Reply:
no strong opinions.  It's understandable. Let's go with 
`get_source_secret_key_from_env` `get_dest_secret_key_from_env`. 👍